pages/products/product-list.html

```diff
@@ -50,7 +50,7 @@
                     <ul class="user-menu">
                         <li><a href="../../pages/cart/cart.html"><i class='bx bx-cart'></i></a></li>
                         <li><a href="../../pages/wishlist/wishlist.html"><i class='bx bx-bookmark'></i></a></li>
```

Comment:
if possible try using absolute paths for consistency

Reply:
Will do it in the next fix, thank you!